libs/community/langchain_community/vectorstores/oraclevs.py

```diff
@@ -520,12 +569,22 @@ def add_texts(
         embeddings = self._embed_documents(texts)
         if not metadatas:
             metadatas = [{} for _ in texts]
-        docs = [
-            (id_, text, json.dumps(metadata), array.array("f", embedding))
-            for id_, text, metadata, embedding in zip(
-                processed_ids, texts, metadatas, embeddings
-            )
-        ]
+
+        docs: List[Tuple[Any, Any, Any, Any]]
```

Comment:
Can you make this type more specific?

Reply:
This was just to avoid lint error. Just below it type was resolved